ansible_collections/arista/avd/roles/eos_designs/python_modules/core_interfaces_and_l3_edge/ethernet_interfaces.py

```diff
@@ -56,6 +56,21 @@ def ethernet_interfaces(self) -> list | None:
                     context_keys=["name", "peer", "peer_interface"],
                 )
 
+        for l3_interface in self._filtered_l3_interfaces:
+            # Ethernet interface
+            ethernet_interface = self._get_l3_interface_cfg(l3_interface)
+
+            # Remove None values
+            ethernet_interface = {key: value for key, value in ethernet_interface.items() if value is not None}
```

Comment:
this should be stripped in the _get_l3_interface_cfg using the strip_none_values stuff.

Reply:
using same logic as p2p link so maybe this is redundant above as well?